Logstash/logstash-input-azuretopicthreadable/lib/logstash/inputs/azuretopicthreadable.rb

```diff
@@ -47,9 +58,12 @@ def process(output_queue, pid)
           codec.decode(message.body) do |event|
           decorate(event)
           output_queue << event
-        end # codec.decode
-        # delete the message after reading it
-        azure_service_bus.delete_subscription_message(message)
+          # delete the message after reading it
+          azure_service_bus.delete_subscription_message(message)
+        end
```

Comment:
@xiaomi7732 Thanks for your suggestion! I didn't see your comment until after I spend some time working though this again today - came to the same conclusion!